README.md

```diff
@@ -136,4 +136,15 @@ Options:
 
 ### `harvester harvest ogm`
 
-_Work in progress, OGM harvests not yet supported from CLI..._
\ No newline at end of file
+```text
+Usage: -c harvest ogm [OPTIONS]
+
+  Harvest and normalize OpenGeoMetadata (OGM) geospatial metadata records.
+
+Options:
+  --include-repositories TEXT  If set, limit to only these comma seperated
```

Comment:
`limit to only *this* comma separated` instead?

Reply:
And same for `--exclude-repositories`

Reply:
Good call!  That language felt clunky.  Updated.